orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/states/SaltStates.java

```diff
@@ -53,6 +54,7 @@
 import com.sequenceiq.cloudbreak.orchestrator.salt.domain.StateType;
 import com.sequenceiq.cloudbreak.service.Retry;
 
+@Service
 public class SaltStates {
```

Comment:
I feel like this should be renamed to `SaltStateService` or similar if it is a service

Reply:
I agree, I did not want to make this step as of now but if you also feel like this is the right naming, then I will change it.

Reply:
thanks, I think it would be nice to have